panels/nerf_panel_operators/import_dataset_operator.py

```diff
@@ -147,7 +157,8 @@ def execute(self, context):
             sensor_width = cam_data.sensor_width
             cam_data.lens = sensor_width / cam_w * fl_x
 
-        context.view_layer.objects.active = nerf_obj
+        context.scene.nerf_panel_ui_props.imported_dataset_path = self.filepath
+        NeRFManager.can_import = False
```

Comment:
I wouldn't mind changing this to a settable value, but right now it's still a method not a property

Reply:
it worked when i was working with it because like you said you can set it to true or false because initially it isn't set to a value. Does this make sense? If it doesn't lmk